lib/enkf/tests/value_export.cpp

```diff
@@ -132,6 +203,28 @@ void test_export_txt() {
     test_assert_double_equal( v2, 200 );
   }
 
+
+  // Export again with more values
+  value_export_append(export_value, "KEY300", "SUBKEY1", 600);
+  test_assert_int_equal( 3 , value_export_size( export_value ));
+  value_export_txt( export_value );
+
+  auto tree = directory_tree("path");
+  test_assert_size_t_equal(tree.size(), 3); // there is also parameters__.txt
+  std::sort(
+      std::begin(tree),
+      std::end(tree),
+      [](std::string const& l, std::string const& r) -> bool {
+          return l.size() < r.size();
+      });
+
+  test_assert_string_equal(tree[0].c_str(), "path/parameters.txt");
+  test_assert_string_equal(
+      tree[2].substr(0, 29).c_str(),
+      "path/parameters.txt_backup_20"); // Fix this in 80 years
```

Comment:
Already looking forward to it 😎

Would it be possible to have a slightly more exact test of this?

Reply:
given how painful it is to deal with time and date in c++, I'd rather wait until we move everything to python (which hopefully will happen before this test start failing :) )